### DISCUSSION TITLE
Style guide?

### DISCUSSION THREAD

**Discussion by hsmalley** [Slackadays/Clipboard#169] [Q&A]
What style guide is expected/wanted for making contributions? 

I noticed some files (for example install.sh) have different styles in the same file, so I was wondering what is preferred. 

If wanted I could make a PR with [pre-commit](https://pre-commit.com/) configured with the desired style(s) too.

**Comment by Slackadays** [comment]
@hsmalley I haven't set up any style guide for the scripts or anything else that isn't C/C++. For the C/C++ code, there are a few clang-format scripts that automatically apply the set style that all that code uses. A good strategy for the scripts is to match whatever you see nearby, but it's no big deal if it differs a little.
